frontend/src/utils/apiWrapper.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+export const getResponseByTractID = tract_id => {
+  /**
+   * Given:
+   * tract id in database
+   *
+   * Returns all field partners associated to that id upon success
+   * Returns GET_TRACT_DATA_FAIL upon failure
+   */
+  let requestString =
+    "http://localhost:5000" + "/response_data?tract_id=" + tract_id;
+  return axios
+    .get(requestString, {
+      headers: {
+        "Content-Type": "application/text"
+      }
+    })
+    .catch(error => {
+      return {
+        type: "GET_TRACT_DATA_FAIL",
+        error
+      };
+    });
+};
+
+export default getResponseByTractID;
```

Comment:
it's really weird that you had to add this at the end but we'll see if we can fix this later